src/blocks/file-blocks/code/index.tsx

```diff
@@ -84,10 +84,12 @@ export default function (props: FileBlockProps) {
 
   const editorRef = React.useRef<HTMLDivElement>(null);
 
-  const viewRef = React.useMemo<{ current: EditorView | null }>(
-    () => ({ current: null }),
-    []
-  );
```

Comment:
turns out this was not necessary, the type of `useRef` is mutable if you call it right 😬